packages/notifications/push/amplify_push_notifications/android/build.gradle

```diff
@@ -65,7 +65,7 @@ android {
 dependencies {
     api "com.google.firebase:firebase-messaging:23.2.0"
     // Import support library for Amplify push utils
-    implementation 'com.amplifyframework:aws-push-notifications-pinpoint-common:2.10.0'
+    implementation 'com.amplifyframework:aws-push-notifications-pinpoint-common:2.11.1'
```

Comment:
Note to reviewers: No new changes were introduced in this package. See: https://github.com/aws-amplify/amplify-android/releases

Reply:
Bumped the other Android deps to 2.11.1 - ideally, these should all be aligned I think (guessing there is the chance for issues if they're not).